packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -4,12 +4,15 @@ import {
   iconClass,
   selectClass,
   selectContainerClass,
+  selectVariants,
 } from './Select.css';
 
 import { SystemIcon } from '@components/Icon';
 import classNames from 'classnames';
 import React, { FC, forwardRef } from 'react';
 
+export type SelectVariants = 'form' | 'transparent';
```

Comment:
I'm not opposed to having a selectVariant type, but as discussed I think it should be based on the actual variant object rather than something that you define yourself. I believe this would be `keyof typeof selectVariants`